packages/block-library/src/home-link/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+import { _x } from '@wordpress/i18n';
+import { home } from '@wordpress/icons';
+import { createInterpolateElement } from '@wordpress/element';
+import { ExternalLink } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit';
+import save from './save';
+
+const { name } = metadata;
+
+export { metadata, name };
+
+export const settings = {
+	icon: home,
+
+	description: createInterpolateElement(
+		_x( 'Link to <a>your homepage</a>', 'block description' ),
+		{ a: <ExternalLink href="options-reading.php" /> }
```

Comment:
I don't think we should use an external link if we are linking within wp-admin. It's also probably worth noting that a static link like this would break in some contexts (clients like Calypso, mobile apps, etc).

Reply:
Good point, @mtias. Is there a recommended pattern for allowing folks to override this value?

Reply:
@Addison-Stavlo noted that we're using a slot/fill pattern in other areas https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-post/src/components/header/main-dashboard-button/index.js https://github.com/WordPress/gutenberg/blob/734e3871c1a9fc1085670a0fc5284baabfdf6b7b/packages/edit-post/src/components/header/index.js#L57-L59

I can maybe try this out too, though I suspect this may need to be generalized for any links in the block settings area.

Reply:
Yes, it can be tricky. It's one where a simpler filter seems better. In any case, I think it's fine not to add the url in this first pass and revisit more generally. We have a few other blocks that want to link to Settings > General or other admin places.